README.md

```diff
@@ -0,0 +1,109 @@
+
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+#  Math Magicians
+
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
+
+
+
+## 🛠Built With
+
+<a href="https://reactjs.org/" target="_blank" rel="noreferrer"> <img
+      src="https://raw.githubusercontent.com/devicons/devicon/master/icons/react/react-original-wordmark.svg"
+      alt="react" width="50" height="50" /> </a>
+
+### Key Features <a name="key-features"></a>
+    - Set up a new project with React and node.js
+    - Use Linters for git hub workflows
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Visual Studio Code
+- Web browser such as chrome
+- Internet Access
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone :  https://github.com/hydaline-code/React.git
+  cd folder 
+  code . :open in vscode
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install : if you have Node.js
+  npm start :Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Live demo
+> A live demo link will be uploaded soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Hydaline**
+
+- GitHub: [https://github.com/hydaline-code]
+- LinkedIn: [https://www.linkedin.com/in/charlene-djoungang-0851aa21a/]
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Apply complete stylings to make more beautiful
+- A complete customization of the SPA
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to microverse
+- Inspiration from other developers 
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please add the valid path for your `LICENSE` file. Your license file path is incorrect, resulting in a 404 page error. Follow the suggestion below to fix the problem and double-check if everything is working without any problem. Having a broken link in your documentation causes a bad impression on the user.
```suggestion
This project is [MIT](./LICENSE) licensed.
```